build.py

```diff
@@ -25,3 +26,5 @@ def set_properties(project):
     project.set_property("flake8_break_build", True)
     project.set_property("flake8_include_test_sources", True)
     project.set_property("flake8_include_scripts", True)
+    # Property for integration test to work on Windows
+    project.set_property("integrationtest_inherit_environment", True)
```

Comment:
Good to link this issue here as a comment for future reference https://github.com/pybuilder/pybuilder/issues/414